frameworks/elastic/src/main/dist/svc.yml

```diff
@@ -289,9 +289,9 @@ pods:
       - {{SCHEDULER_URI}}
       - {{BOOTSTRAP_URI}}
     tasks:
-      server:
+      node:
```

Comment:
Yeah, I debated on that. It certainly makes more sense as a server (and it's a singleton). I'll change it back.